tests/core/test_core.py

```diff
@@ -419,7 +463,9 @@ def set_msg_ids(*args, **kwargs):
             num_after=0, num_before=30, anchor=10000000000
         )
         create_msg.assert_called_once_with(controller.model, msg_ids)
-        assert controller.model.index == {"search": msg_ids}
+        assert controller.model.index == dict(
+            index_search_messages, **{"search": msg_ids}
+        )
```

Comment:
This was originally a mypy error when I was making an assignment like this to `controller.model.index` where mypy mentioned that the types did not match (`Index` and `Dict[str, object]`); I don't think mypy catches type errors when the only interaction ever being made is an `assert`.